graviti/openapi/data.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 Graviti. Licensed under MIT License.
+#
+
+"""Interfaces about the data."""
+from typing import Any, Dict, Optional
+from urllib.parse import urljoin
+
+from graviti.openapi.requests import open_api_do
+
+
+def list_draft_data(
+    url: str,
+    access_key: str,
+    owner: str,
+    dataset: str,
+    *,
+    draft_number: int,
+    sheet: str,
+    columns: Optional[str] = None,
+    order_by: Optional[str] = None,
+    offset: int = 0,
+    limit: int = 128,
+) -> Dict[str, Any]:
+    """Execute the OpenAPI`GET /v2/datasets/{owner}/{dataset}/drafts/{draft_number}\
+    /sheets/{sheet}/data`.
+
+    Arguments:
+        url: The URL of the graviti website.
+        access_key: User's access key.
+        owner: The owner of the dataset.
+        dataset: Name of the dataset, unique for a user.
+        draft_number: The draft number.
+        sheet: The sheet name.
+        columns: The string of column names separated by commas.
+            Multiple indexes can be expressed using dots. None means to get all columns.
+        order_by: The string of column names separated by commas.
```

Comment:
do we have defined the behavior of `order_by`? @graczhual

Reply:
ben add the description about order_by in the doc, here the translation here is not very accurate